### ISSUE TITLE
luarocks not found

### ISSUE THREAD

**Issue by chenxuefeng1207** [Kong/docker-kong#247]
I use kong:1.1.1-alpine this image and run
`docker run -it --rm --privileged kong:1.1.1-alpine sh `
then I run luarocks but got :
`/ # luarocks
sh: luarocks: not found`
I found luarocks-5.1 in /usr/local/bin but I run `luarocks-5.1` also the same error.

**Comment by hutchic** [comment]
yup the `docker-kong` test to verify that luarocks is installed / works correctly was incorrect and I believe this should fix that so we don't have this happen again https://github.com/Kong/docker-kong/pull/249

as for the root cause issue that fix is PR'd here and also includes a test https://github.com/Kong/kong-build-tools/pull/38

using the alpine asset from https://bintray.com/kong/kong-community-edition-alpine-tar/ will work or running `sed -i 's/\/tmp\/build//' /usr/local/bin/*` until we fix the images properly are the workarounds available

**Comment by igorbrites** [comment]
The `sed` commando solved the path to `luarocks`, but my plugin depends on `lua-requests`, and luarocks still looks for the dependencies on `/tmp/build` as follows:

```

**Comment by hutchic** [comment]
I double checked the pending update to the `1.1.1` image which can be built locally from https://github.com/Kong/docker-kong/tree/1.1.1 until docker merges https://github.com/docker-library/official-images/pull/5658 works

```
git clone git@github.com:Kong/docker-kong.git
cd docker-kong
git checkout 1.1.1
cd alpine
docker build -t kong:1.1.1 .
docker run -it --rm --entrypoint=/bin/sh kong:1.1.1
apk add --no-cache gcc libc-dev g++ util-linux
luarocks install lua-requests 1.0-0
```

**Comment by hutchic** [comment]
This should be resolved in the `1.1.1` ( `latest` ) images now

**Comment by igorbrites** [comment]
Thank you a lot @hutchic! Works like a charm!
